### ISSUE TITLE
Bug: error: unsupported tarball input attribute 'rev'

### ISSUE THREAD

**Issue by dr460nf1r3** [garnix-io/issues#76]
Hey! Thanks for this amazing project. Been using it for a few weeks!

I'm getting the [following error since today](https://garnix.io/build/jgNMjjz9): 
~~~
Command 'nix flake metadata --json' failed with exit code 1. Stderr:
error: unsupported tarball input attribute 'rev'
(use '--show-trace' to show detailed location information)
~~~
The same command on the same commit clearly works locally:
~~~
❯ nix flake metadata --json | jq                                                                                                                                                                     
{                                                                                                                                                                                                    
  "description": "Dr460nixed NixOS flake ❄️",                                                                                                                                                         
  "lastModified": 1695404317,                                                                                                                                                                        
  "locked": {                                                                                                                                                                                        
    "lastModified": 1695404317,                                                                                                                                                                      
    "narHash": "sha256-82EzxDWrjfGcuFD47LMuu6An3ZNSB9TgtozN4JvZSmQ=",                                                                                                                                
    "ref": "refs/heads/main",                                                                                                                                                                        
    "rev": "182ea234c163c041e95a6ed511a4608562593888",                                                                                                                                               
    "revCount": 666,                                                                                                                                                                                 
    "type": "git",                                                                                                                                                                                   
    "url": "file:///home/nico/Documents/misc/dr460nixed"
~~~

**Comment by dr460nf1r3** [comment]
What Nix version is used to build the derivations? Seems it is caused by using an older version of it:
https://flakehub.com/docs#i-got-error-unsupported-tarball-input-attribute-rev-what-does-this-error-mean

**Comment by ajaxbits** [comment]
> What Nix version is used to build the derivations?

From the discord, it was `2.11.1` on 9/25/23

**Comment by jkarni** [comment]
Nix has been upgraded to `2.17` now!

**Comment by LudovicoPiero** [comment]
I encountered a similar error, but this time it's 'lastModified' instead of 'rev'

```console
Command 'nix flake metadata --json' failed with exit code 1. Stderr:


error:
       … while updating the lock file of flake 'git+file:///tmp/garnix-runs-7b9815a9865a5d1c?ref=de55f8a40dae8bc1a23c97f018eaf838b54b1931&rev=de55f8a40dae8bc1a23c97f018eaf838b54b1931'

       error: unsupported tarball input attribute 'lastModified'
```

**Comment by dr460nf1r3** [comment]
This has been an issue since today (?), at least I noticed as of today. Cross-posting from Discord:
> Hello @dr460nf1r3 , indeed we are running 2.17 and this seems to be fixed in 2.17.1 (cf https://github.com/NixOS/nix/commit/eb21904198e395c4068543ddbfbd10db4c80be46)
I will see with the team when we can update nix and keep you informed !

And I can already confirm that the deployed update to 2.17.1 made things work again.
